.github/workflows/ci.yml

```diff
@@ -79,3 +105,47 @@ jobs:
         export PATH=$PATH:/c/Users/runneradmin/AppData/Roaming/local/bin;
         bats ./tests/golden/**
       shell: bash
+
+  xrefcheck-build-and-test-nix:
+    runs-on: [self-hosted, nix]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
```

Comment:
What does it mean? Do we have submodules?

Reply:
Yes, several libraries for bats:
https://github.com/serokell/xrefcheck/tree/master/tests/golden/helpers

Reply:
Oh, ok.